src/components/QuestDetails/index.tsx

```diff
@@ -140,6 +149,11 @@ export default function QuestDetails({
     )
   }
 
+  const isEndDateInFuture = endDate && new Date(endDate) > new Date()
+  const utcFormattedDate = dayjs(endDate)
+    .utc()
+    .format('MM/DD [at] HH:mm [(UTC)]')
```

Comment:
hmm why do we want to show this date time in UTC? shouldn't we show it in the user's timezone @pimentel08 @iamdavid0091 ?

Reply:
also can we parse into a string format that fits the user's locale? most countries use DD/MM (day/month) date format but US is MM/DD

Reply:
great call, I found out we can use dayjs for this https://day.js.org/docs/en/display/format#localized-formats

I decided to use the `lll` format.


<img width="393" alt="image" src="https://github.com/user-attachments/assets/458d9f7f-dda1-4dbd-9c73-bf2af43fd9db" />

Reply:
Design agreed with using the localized `lll` format in the users timezone